packages/Liquid-Tests.package/LiquidPollTests.class/instance/testClosePoll.st

```diff
@@ -26,7 +26,7 @@ testClosePoll
 				 pollId: poll id;
 				 yourself.
 	poll addAnswerSet: a1;
-		 isOpen: false;
+		 closeWithPassword: nil;
```

Comment:
I assume we could use anything here, since the `LQGuard` is an `LQAcceptingGuard`? 
[pebble] In my opinion, since closing a poll got quite a bit more complex, we should actually start the poll to give it a password and test the complete process of closing a Poll, instead of skipping half of what would normally need to happen.